README.md

```diff
@@ -23,7 +23,7 @@ Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run/?git_repo=
     [![Run on Google Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run)
     ```
 
-1. If the repo contains a `Dockerfile`, it will be built using the `docker build` command.  If the repo uses Maven for the build and it contains the [Jib plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin), then the container image will be built with Jib ([Jib Spring Boot Sample](https://github.com/GoogleContainerTools/jib/tree/master/examples/spring-boot)).  Otherwise, [CNCF Buildpacks](https://buildpacks.io/) (i.e. the `pack build` command) will attempt to build the repo ([buildpack samples][buildpack-samples]).
+1. If the repo contains a `Dockerfile`, it will be built using the `docker build` command.  If the repo uses Maven for the build and it contains the [Jib plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin), then the container image will be built with Jib ([Jib Spring Boot Sample](https://github.com/GoogleContainerTools/jib/tree/master/examples/spring-boot)).  Otherwise, [CNCF Buildpacks](https://buildpacks.io/) (i.e. the `pack build` command) will attempt to build the repo ([buildpack samples][buildpack-samples]).  Alternatively the build command can be defined in the `app.json` - see below.
```

Comment:
> Alternatively the build command can be defined in the `app.json` - see below.

This looks outdated.

> Alternatively, you can skip build using the `build.skip` field (see below) and use a "prebuild" or "postbuild" hook to build and push the container image yourself.

Also we can paragraphize this entire bullet point like:

```
1. lorem ipsum

   lorem ipsum 

   lorem ipsum

1. another item
```